DescribeEndpoints* functions are missing

Using the DAX Go client with the latest version of the Go SDK results in:

```
projects/src/github.com/aws/aws-dax-go/dax/service.go:129:5: cannot use (Dax)(nil) (type Dax) as type dynamodbiface.DynamoDBAPI in assignment:
*Dax does not implement dynamodbiface.DynamoDBAPI (missing DescribeEndpoints method)
```

The DescribeEndpoints, DescribeEndpointsWithContext, and DescribeEndpointsRequest functions are missing from api.go.

--- a/dax/api.go
+++ b/dax/api.go
@@ -346,6 +346,19 @@ func (d *Dax) DescribeContinuousBackupsRequest(*dynamodb.DescribeContinuousBacku
 	return
 }
 
+func (d *Dax) DescribeEndpoints(*dynamodb.DescribeEndpointsInput) (*dynamodb.DescribeEndpointsOutput, error) {
+	return nil, d.unImpl()
+}
+
+func (d *Dax) DescribeEndpointsWithContext(aws.Context, *dynamodb.DescribeEndpointsInput, ...request.Option) (*dynamodb.DescribeEndpointsOutput, error) {
+	return nil, d.unImpl()
+}
+
+func (d *Dax) DescribeEndpointsRequest(*dynamodb.DescribeEndpointsInput) (r *request.Request, o *dynamodb.DescribeEndpointsOutput) {
+	d.unImpl()
+	return
+}
+
 func (d *Dax) DescribeGlobalTable(*dynamodb.DescribeGlobalTableInput) (*dynamodb.DescribeGlobalTableOutput, error) {
 	return nil, d.unImpl()
 }

